Mathlib/Topology/Order/UpperLowerSetTopology.lean

```diff
@@ -441,18 +441,18 @@ def map (f : α →o β) : C(WithUpperSetTopology α, WithUpperSetTopology β) w
 @[simp] lemma map_comp (g : β →o γ) (f : α →o β): map (g.comp f) = (map g).comp (map f) := rfl
 
 @[simp] lemma toUpperSet_specializes_toUpperSet {a b : α} :
-  toUpperSet a ⤳ toUpperSet b ↔ b ≤ a := by
+    toUpperSet a ⤳ toUpperSet b ↔ b ≤ a := by
   simp_rw [specializes_iff_closure_subset, UpperSetTopology.closure_singleton, Iic_subset_Iic,
     toUpperSet_le_iff]
 
 @[simp] lemma ofUpperSet_le_ofUpperSet {a b : WithUpperSetTopology α} :
-  ofUpperSet a ≤ ofUpperSet b ↔ b ⤳ a := toUpperSet_specializes_toUpperSet.symm
+    ofUpperSet a ≤ ofUpperSet b ↔ b ⤳ a := toUpperSet_specializes_toUpperSet.symm
```

Comment:
Does the style guide have anything to say here? I'm not sure what the convention is supposed to be.

Reply:
[Yep](https://leanprover-community.github.io/contribute/style.html#structuring-definitions-and-theorems): "If the theorem statement requires multiple lines, indent the subsequent lines by 4 spaces."